tests/acceptance/features/bootstrap/UserLdapGeneralContext.php

```diff
@@ -25,6 +25,7 @@
 use Behat\Gherkin\Node\PyStringNode;
 use Behat\Gherkin\Node\TableNode;
 use Behat\MinkExtension\Context\RawMinkContext;
+use PHPUnit\Framework\Assert;
```

Comment:
Do we need to import `Assert` namespace?

Reply:
good catch, fixed